src/main/java/uk/gov/ons/ctp/response/casesvc/service/CaseService.java

```diff
@@ -156,10 +156,13 @@ public Case findCaseByIac(final String iac) throws CTPException {
    * @param partyId the partyId
    * @return the cases for the partyId
    */
-  public List<Case> findCasesByPartyId(final UUID partyId) {
+  public List<Case> findCasesByPartyId(final UUID partyId, boolean iac) {
     log.debug("Entering findCasesByPartyId");
     List<Case> cazes = caseRepo.findByPartyId(partyId);
```

Comment:
Is this named `cazes` to avoid clash with another `cases` variable in scope?

Reply:
no, it's named `cazes` because when we have an object of class `Case` we can't name it `case` because `case` is a reserved word... the convention for the name of a collection of `Case` objects has thus become `cazes` which is perhaps something which could be addressed in future, but isn't really in the scope of this pull request, although I appreciate the gravity of the situation of having several collections named `cazes` and the urgent need to change it everywhere it occurs.